ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/baselinecaps/vcurrent/provision/cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-baselinecaps-vcurrent-provision-chain
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
```

Comment:
missing `additional-enabled-capabilities`?

Reply:
baselineCapabilitySet is set to vCurrent, it will install all optional operators in set of cluster's current version, no more additional operators to be installed, so no need to add step additional-enabled-capabilities in this chain.

Reply:
I saw that, the baselineCapabilitySet is set in `ipi-conf`, sorry for confused comment.